src/config/routes/2.1.json

```diff
@@ -144,6 +144,13 @@
     },
 
     {
+        "path": "/users(/[^/]+)*/?$",
+        "controller": "UsersController",
+        "action": "deleteAction",
```

Comment:
Sorry to prove you wrong, but have a look a few lines down... ```action: updateUser```

And I had a discussion some time ago with Lorna and Rob and at least at that time we agreed on making the actions more "speaking" (at least that's what I remember)